script/Arcueid/Day 5/05_00_ARC05_9_3.txt

```diff
@@ -183,7 +183,8 @@ It's not that Vlov didn't %{i}want%{/i} to approach us.
 [sha:54bfa73523fa85dc6b44ace59002f8df421a35e7]{
 -- Page 813, Offset 10866.
 -- だというのに、俺はヤツに追い風を与えてしまった。
-And in spite of that that, I gave him a tailwind.
+// did that that get past the lint?
```

Comment:
not sure what you're asking here

Reply:
it's "how did that that get past the lint". I ruined the joke by forgetting the first word

Reply:
second one has a comma on it, so it's a different 'word'